components/ai_chat/resources/page/components/conversations_list/index.tsx

```diff
@@ -62,26 +62,36 @@ interface DisplayTitleProps {
 }
 
 function DisplayTitle(props: DisplayTitleProps) {
-  const [isButtonMenuVisible, setIsButtonMenuVisible] = React.useState(false)
+  const [isOptionsMenuOpen, setIsOptionsMenuOpen] = React.useState(false)
+
+  const handleButtonMenuChange = React.useCallback((e: {isOpen: boolean}) => {
```

Comment:
optional: with stuff like this where you're at a leaf node, I think its okay to not wrap things in a `useCallback` (or do it inline)

Up to you on this one

Reply:
I just wasn't sure if our web component would cause a new listener to be added (and the old one removed) on every single render. But I suppose we're doing that everywhere anyway

Reply:
It will :laughing: - but its the same with every other component that does this so :shrug: